skyportal/handlers/api/photometry.py

```diff
@@ -757,13 +794,23 @@ def put(self):
                 # update the corresponding photometry entry in the db
                 duplicate.groups = groups
 
+            # posting to new streams?
+            if stream_ids:
+                streams = Stream.get_if_accessible_by(stream_ids, self.current_user)
+                # update the corresponding photometry entry in the db
+                duplicate.streams.extend(streams)
```

Comment:
I don't think this actually fixes the issue, unfortunately. Accessing `duplicate.streams` to `extend` it will still load all streams into the session. I tried writing a test for this [here](https://github.com/kmshin1397/skyportal/commit/55f7220621bf0f288fcaa1d33ece3e3c49afd7ae) and saw the "Insufficient permissions to read stream" message